python/mxnet/ndarray.py

```diff
@@ -403,12 +403,26 @@ def __getitem__(self, key):
 
 
     def _sync_copyfrom(self, source_array):
-        """Peforms a synchronized copy from the array.
+        """Performs a synchronized copy from the source_array to the current array.
+        This is called through x[:] = source_array, where the source_array
+        is a numpy.ndarray or array_like object.
```

Comment:
numpy.ndarray should be in single backticks (clas) 

"array_like" -> "array-like"